br/tests/lightning_issue_34208/config.toml

```diff
@@ -0,0 +1,2 @@
+[tikv-importer]
```

Comment:
prefer we name the test directory more clearly, it's hard to know what's inside at first glance